_includes/candidate-summary.html

```diff
@@ -6,11 +6,13 @@
     <img class="candidate-summary__photo" src="{{ candidate.photo_url | default: 'https://s3-us-west-1.amazonaws.com/odca-candidate-photos/no-image.png' }}" alt="{{ candidate.name | smartify | escape }}" />
   </div>
   <div class="candidate-summary__right">
-    <div><a href="{{ candidate.url | prepend: site.baseurl }}">{{ candidate.name | smartify | escape }}</a></div>
-    <div class="candidate-summary__occupation">{% if candidate.is_incumbent %}Incumbent, {% endif %}{{ candidate.occupation | default: 'No occupation reported' | escape }}</div>
-    {% if finance.total_contributions > 0 %}
+    <div class="candidate-summary__name"><a href="{{ candidate.url | prepend: site.baseurl }}">{{ candidate.name | smartify | escape }}</a></div>
+    <div class="candidate-summary__occupation">{% if candidate.is_incumbent %}Incumbent {% endif %}{{ candidate.occupation | default: 'No occupation reported' | escape }}</div>
+    {% if finance.total_contributions > 2000 %}
       <div class="candidate-summary__contributions">amount collected <span class="money">{{ finance.total_contributions | dollars }}</span></div>
-    {% else %}
+    {% elsif finance.total_contributions > 0 and finance.total_contributions < 2000 %}
```

Comment:
I think this can just be an `else` and we should delete the last condition.

The $2000 rule means that they don't have to file until they hit that $2000 mark. If we have no data on the candidate (0 contributions) it really means they haven't hit that $2000 mark. I don't think we should ever be saying "no contributions" here, instead we should just say that they have not raised $2000. @sfdoran can you clarify this?

Reply:
@adborden There are 3 scenarios where a candidate doesn't have contribution or expenditure data:

1) They do not intend to raise or spend $2,000 or more and are therefore not required to register their committee and file campaign statements.  We know which candidates these are because they do not have a campaign committee with a filer name or a filer ID number. So a candidate can be in compliance and not have any campaign finance data.

2) The candidate has registered their committee and raised/spent over $2,000, but they have not yet been required to file - e.g. the first deadline for most of our candidates is 7/31 for 1/1 - 6/30/2018 unless they started fundraising in 2017. If they start their committee after 6/30, they may not show data until the pre-election deadline in September or when they file a late contribution report (Form 497). So again, they could be in compliance but not show any data.

3) The candidate has registered their committee but failed to file a campaign statement, i.e. they are out of compliance.

If you can indicate candidates without registered committees (those without filer IDs) versus those with registered committees, but no campaign finance data I think that would be good. I don't want people visiting the site to think no data means the candidate is in violation.

Reply:
Thanks for the clarificaiton, let's move this convo to #163 and track it there.

Reply:
@jbotwins for this PR, let's go back to the original `if contributions > 0` check, and update the copy in the `else` to be "no contribution data has been reported"

Reply:
Thanks for coordinating Aaron! Thanks for clarifying Suzanne!

I will implement what Aaron notes in his last comment.

"if contributions > 0" the number displays.

If that is not true the copy will be "No contribution data has been reported."